tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -46,217 +38,6 @@
 DEFAULT_BUCKET_COUNT = 30
 
 
-def histogram_v2(name, data, step=None, buckets=None, description=None):
```

Comment:
Now that we're removing `histogram_v2`, I think it'd make sense to rename `histogram_v3` to just `histogram` (what the v2 one was called prior to the migration). We can then remove the `histogram = histogram_v3` line at the bottom.

When we do that, it might make sense to add a `#`-style comment (not in the docstring, which is public, but just for our own reference) that mentions that this new `histogram` is actually an updated v3 format, not the v2 format, as the filename would imply.

Reply:
Renamed and added the comment.